package.json

```diff
@@ -33,15 +30,62 @@
     "JSONStream": "1.3.1"
   },
   "devDependencies": {
-    "eslint": "^4.1.1",
+    "commitizen": "^2.9.6",
+    "cz-conventional-changelog": "^2.1.0",
+    "eslint": "^4.10.0",
     "eslint-config-finn": "^2.0.0",
-    "tap": "10.7.0"
+    "eslint-config-finn-prettier": "^3.0.1",
+    "husky": "^0.14.3",
+    "jest": "^21.2.1",
+    "lint-staged": "^4.3.0",
+    "prettier": "^1.8.1",
+    "projectz": "^1.4.0"
   },
   "scripts": {
     "lint": "eslint .",
-    "test": "tap test/*.js"
+    "test": "jest --coverage",
+    "lint:format": "eslint --fix .",
+    "precommit": "lint-staged",
+    "cm": "git-cz",
+    "readme": "projectz compile"
   },
   "files": [
     "lib"
-  ]
+  ],
+  "jest": {
+    "testEnvironment": "node",
+    "clearMocks": true,
+    "coverageThreshold": {
+      "global": {
+        "branches": 26,
+        "functions": 14,
+        "lines": 29,
+        "statements": 29
+      }
+    }
+  },
+  "lint-staged": {
+    "*.js": [
+      "eslint --fix --config ./.eslintrc",
+      "git add"
+    ],
+    "{package.json,README.md,LICENSE.md}": [
```

Comment:
pass them through `prettier --write` as well :D

Reply:
fixed.

Reply:
@digitalsadhu It'd be great if a `prettier` section was added by default

Reply:
cool adding to boring

Reply:
probably want to scope it to just the readme though as the `"eslint --fix --config ./.eslintrc"` in the js section is using prettier with our eslint rules. Prettier by itself will screw up the js files right?

Reply:
prettier --write README.md LICENSE.md package.json

Reply:
prettier should just be set up with out rules. No need for prettier on js files as a commit-hook, though

Reply:
```json
  "prettier": {
    "singleQuote": true,
    "trailingComma": "all",
    "tabWidth": 4,
    "overrides": [
      {
        "files": "*.json",
        "options": {
          "tabWidth": 2
        }
      },
      {
        "files": ".eslintrc",
        "options": {
          "parser": "json",
          "tabWidth": 2
        }
      }
    ]
  }
```

Reply:
yeah that looks good

Reply:
This look ok?
https://github.com/digitalsadhu/boring/blob/master/templates/package.json#L22-L49

Reply:
Should drop glob from https://github.com/digitalsadhu/boring/blob/master/templates/package.json#L46

Reply:
Other than that, perfect!

Reply:
hmm. Think I'm not quite getting it. If I drop the globbing won't prettier run on everything (including the js files)?

Reply:
No, lint staged passes on the files matching its glob which was staged in git. It never runs stuff on all files

Reply:
Ahhhhh
Knew I was missing something!